python/hail/representation/variant.py

```diff
@@ -433,26 +460,32 @@ class Locus(HistoryMixin):
     :param contig: chromosome identifier
     :type contig: str or int
     :param int position: chromosomal position (1-indexed)
+    :param reference_genome: Reference genome to use. Default is :class:`~.HailContext.default_reference`.
+    :type reference_genome: :class:`.GenomeReference`
     """
 
     @handle_py4j
     @record_init
-    def __init__(self, contig, position):
+    @typecheck_method(contig=oneof(strlike, integral),
+                   position=integral,
+                   reference_genome=nullable(GenomeReference))
+    def __init__(self, contig, position, reference_genome=None):
         if isinstance(contig, int):
             contig = str(contig)
         jrep = scala_object(Env.hail().variant, 'Locus').apply(contig, position)
         self._init_from_java(jrep)
         self._contig = contig
         self._position = position
+        self._rg = reference_genome if reference_genome else Env.hc().default_reference
```

Comment:
can you add an accessor method for the reference genome to locus, variant, interval?

Reply:
Isn't that what Variant.reference_genome() is, or are you talking about something else?

Reply:
oh, wow. Yes, don't know how I missed that! Must have been before I started munching espresso beans.

Reply:
I made it a property (also treat property methods different than regular methods in the history). So it's `v = Variant(...); v.reference_genome`